static/src/javascripts-legacy/projects/common/modules/discussion/api.js

```diff
@@ -26,22 +26,26 @@ define([
      * @return {Reqwest} a promise
      */
     Api.send = function (endpoint, method, data) {
-        data = data || {};
+        if (config.switches.enableDiscussionSwitch) {
```

Comment:
@nicl  This is just future proofing the JS incase a feature is added that might execute this code and doesn't live in the discussionFooter html, what do you think?